cdap-common/src/test/java/co/cask/cdap/common/lang/jar/BundleJarUtilTest.java

```diff
@@ -46,6 +44,14 @@
   @ClassRule
   public static final TemporaryFolder TEMP_FOLDER = new TemporaryFolder();
 
+  @Test
+  public void test() throws Exception {
```

Comment:
was this test case intended? Looks like a sanity check

Reply:
Oops.. forget to remove.